receiver/mongodbatlasreceiver/metadata.yaml

```diff
@@ -17,6 +17,10 @@ resource_attributes:
     description: Hostname of the process
     enabled: true
     type: string
+  mongodb_atlas.user.alias:
```

Comment:
I found a more detailed description [here](https://www.mongodb.com/docs/atlas/reference/api/processes-get-one/#response-elements): `User-friendly hostname of the cluster node. The user-friendly hostname is typically the standard hostname for a cluster node and it appears in the connection string for a cluster instead of the value of the hostname field.`

If I'm understanding correctly, this is an alternate value to the existing `mongodb_atlas.host.name` attribute. Does it make sense to collect both? Should we have a flag that allows the user to select which one they want?

Reply:
If we keep this, I suggest we change it to `mongodb_atlas.host.alias` because it is an alias that refers to the host. Being user friendly is a characteristic of the value, but not the meaning of it.

Reply:
Thanks for reviewing this @djaglowski 

> If I'm understanding correctly, this is an alternate value to the existing mongodb_atlas.host.name attribute. Does it make sense to collect both? Should we have a flag that allows the user to select which one they want?

Yeah, this is my understanding top. My goal was to find the same attribute the user was seeing in the UI after they reported the discrepancy between  `mongodb_atlas.host.name` and what the portal was showing and the `userAlias` was the answer.

> If I'm understanding correctly, this is an alternate value to the existing mongodb_atlas.host.name attribute. Does it make sense to collect both? Should we have a flag that allows the user to select which one they want?

Yes, I am keeping both `hostname` and `userAlias`, not only for backward compatibility, but user stated that the both attributes are useful for them.

> if we keep this, I suggest we change it to mongodb_atlas.host.alias because it is an alias that refers to the host. Being user friendly is a characteristic of the value, but not the meaning of it.

I chose `mongodb_atlas.host.alias` to keep the same pattern in place and to match what the naming the API is returning (ie: userAlias); especially that this field is documented internally and they can cross reference it with mongodb API docs.

Reply:
I guess I could go with `mongodb_atlas.host.user_alias` - just so we get a shade of blue on that bikeshed.

Reply:
> I guess I could go with `mongodb_atlas.host.user_alias` - just so we get a shade of blue on that bikeshed.

That seems reasonable.